Remotion/Web/Core/ExecutionEngine/WxePageInfo.cs

```diff
@@ -233,9 +232,7 @@ private bool FindHtmlFormControlFilter (MemberInfo member, object? filterCriteri
     }
     private void HandlePageInit (object? sender, EventArgs e)
     {
-      var resourceUrlFactory = SafeServiceLocator.Current.GetInstance<IResourceUrlFactory>();
-      var scriptUrl = resourceUrlFactory.CreateResourceUrl(typeof(WxePageInfo), ResourceType.Html, c_scriptFileUrl);
-      HtmlHeadAppender.Current.RegisterJavaScriptInclude(s_scriptFileKey, scriptUrl);
+      HtmlHeadAppender.Current.RegisterWebClientScript();
```

Comment:
```suggestion
```

Redudnant. this and the second part with the styleUrl should get moved to `OnPreRenderComplete`. It needs testing that things still work.